### ISSUE TITLE
Search in multiple Chrome profiles

### ISSUE THREAD

**Issue by emirhan87** [Acidham/chromium-hist-bookmarks#37]
Most people use multiple Chrome profiles (work, home, etc.) and it would make sense to search in all bookmarks and all history.

**Comment by zhanzizhen** [comment]
same problem. 
I solved the problem by editing the config file which is located in /Users/xxxx/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.xxxx/chrom_history.py:
![image](https://github.com/Acidham/chromium-hist-bookmarks/assets/22932241/d609b778-932f-48eb-9882-560819b1962a)

**Comment by zhanzizhen** [comment]
> same problem. I solved the problem by editing the config file which is located in /Users/xxxx/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.xxxx/chrom_history.py: ![image](https://private-user-images.githubusercontent.com/22932241/346902470-d609b778-932f-48eb-9882-560819b1962a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA1MTc2NjUsIm5iZiI6MTcyMDUxNzM2NSwicGF0aCI6Ii8yMjkzMjI0MS8zNDY5MDI0NzAtZDYwOWI3NzgtOTMyZi00OGViLTk4ODItNTYwODE5YjE5NjJhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA5VDA5MjkyNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNhYzc4Mzk1ZTIwNmFmMzQ4MmEyZThhMjQ1MTgwOThlMzc0NTE5NTNjNjZhZGJiN2ViNmFiNTdjZDY1M2Y5OTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.UAXSIJjUct4yNRWZbw5YENMawVCM9bZy4YEBOtCL9SA)

This workflow doesn't work if multiple Chrome profiles are used because the path "Library/Application Support/Google/Chrome/Default/History" doesn't exist.

**Comment by Acidham** [comment]
yep, per design, but the workaround is a valid workaround. I can think of making the path configurable 🤔

**Comment by zyfloid** [comment]
To use more than one edge profile either in bookmark-search or in history search simply add another "edge"-entry. For instance, this are my entries in the py-files:

chrom_history.py:
HISTORY_MAP = {
    "brave": "/Library/Application Support/BraveSoftware/Brave-Browser/Default/History",
    "brave_beta": "/Library/Application Support/BraveSoftware/Brave-Browser-Beta/Default/History",
    "chromium": "/Library/Application Support/Chromium/Default/History",
    "chrome": "/Library/Application Support/Google/Chrome/Default/History",
    "opera": "/Library/Application Support/com.operasoftware.Opera/History",
    "sidekick": '/Library/Application Support/Sidekick/Default/History',
    "vivaldi": "/Library/Application Support/Vivaldi/Default/History",
    "edge": "/Library/Application Support/Microsoft Edge/Default/History",
    "edge": "/Library/Application Support/Microsoft Edge/Profile 1/History"
}



chrom_bookmarks.py:
BOOKMARS_MAP = {
    "brave": '/Library/Application Support/BraveSoftware/Brave-Browser/Default/Bookmarks',
    "brave_beta": '/Library/Application Support/BraveSoftware/Brave-Browser-Beta/Default/Bookmarks',
    "chrome": '/Library/Application Support/Google/Chrome/Default/Bookmarks',
    "chromium": '/Library/Application Support/Chromium/Default/Bookmarks',
    "opera": '/Library/Application Support/com.operasoftware.Opera/Bookmarks',
    "sidekick": '/Library/Application Support/Sidekick/Default/Bookmarks',
    "vivaldi": '/Library/Application Support/Vivaldi/Default/Bookmarks',
    "edge": '/Library/Application Support/Microsoft Edge/Default/Bookmarks',
    "edge": '/Library/Application Support/Microsoft Edge/Profile 1/Bookmarks'
}

So look in your home directory//Library/Application Support/Microsoft Edge/ for further Folders like "Profile 1", "Profile 2" etc.

@Acidham thank you very much for this great workflow!
